README.md

```diff
@@ -94,6 +94,16 @@ max-login-attempts-starter:
   clear-all-attempts-cron: '0 0 0 25 12 *' # Christmas time at 00:00
 ```
 
+## Enabling the reset of attempts by username after specified ms 
```

Comment:
Is dit niet hetzelfde als het `Overriding the default after-blocking cooldown` die hierboven staat uitgelegd?

Reply:
Nee, die cron reset van iedereen de blocked status en attempts. De clear-attempts doet het voor één username en alleen de attempts